pkg/system/phase2_creating.go

```diff
@@ -560,6 +560,28 @@ func (r *Reconciler) SetDesiredCoreApp() error {
 				}}
 				util.MergeVolumeMountList(&c.VolumeMounts, &secretVolumeMounts)
 			}
+		case "noobaa-log-processor":
+			if c.Image != r.NooBaa.Status.ActualImage {
+				coreImageChanged = true
+				c.Image = r.NooBaa.Status.ActualImage
+			}
+			// adding the missing Env variable from default container
+			util.MergeEnvArrays(&c.Env, &r.DefaultCoreApp.Env)
+			r.setDesiredCoreEnv(c)
+			r.setDesiredRootMasterKeyMounts(podSpec, c)
+
+			if r.NooBaa.Spec.CoreResources != nil {
+				c.Resources = *r.NooBaa.Spec.CoreResources
```

Comment:
Don't use core resources for the logs container. we should probably make it configurable through noobaa CR (optionally param, same as `CoreResources`), but for now you can just use a constant amount (200m, 500MB)

Reply:
Did you mean we should use hard code values 200m and 500MI in these lines?I have made changes. Please check if this is what you meant.

Reply:
@aspandey, yes this is what I meant. I see now that the tests are failing due to resources, because with `--mini` we use `100m` for core.  let's add `LogResources`, to noobaa spec, the same as we have [for core](https://github.com/dannyzaken/noobaa-operator/blob/76c01ce9669a6206ddf7224342e3a26e88423b23/pkg/apis/noobaa/v1alpha1/noobaa_types.go#L93-L96), where the default is `200m`. for [mini env](https://github.com/dannyzaken/noobaa-operator/blob/76c01ce9669a6206ddf7224342e3a26e88423b23/pkg/system/system.go#L215-L243) let's set it to `100m` or some other value that will make the tests pass